mkdocs.yml

```diff
@@ -64,6 +64,7 @@ nav:
         - Triggers: './microservices/application/Triggers.md'
         - './microservices/application/ApplicationFunctionsSDK.md'
         - './microservices/application/AppServiceConfigurable.md'
+        - './microservices/application/Configuration.md'
```

Comment:
Move to be sub section of app service configurable

Reply:
It isn't specific to ASC